atomate/vasp/firetasks/write_inputs.py

```diff
@@ -289,34 +290,49 @@ def run_task(self, fw_spec):
 
 
 @explicit_serialize
-class UpdateScanRelaxBandgap(FiretaskBase):
+class WriteScanRelaxFromPBE(FiretaskBase):
```

Comment:
I wonder if this could be called something else because PBE isn't used. What about `WriteScanRelaxFromGGA`?

Reply:
Done. `WriteScanRelaxFromPrev`.